### DISCUSSION TITLE
LazyExtras->Copilot-Chat - breaking changes?

### DISCUSSION THREAD

**Discussion by grappas** [LazyVim/LazyVim#4974] [General]
I liked a feature \<leader\>ad where prompt would explain lsp diagnostics on current line while being aware of entirity of a buffer. Now it has been removed. Explain from prompt actions or CopilotChatFixDiagnostic doesn't do anything close to it.  Why?

**Comment by folke** [comment]
Why ask here? Maybe check in the copilot chat repo?

**Reply by grappas** [reply]
Aren't keybindings defined in LazyVim? They haven't been changed without reason and I'm asking for it.

**Reply by dpetka2001** [reply]
Here is the [commit](https://github.com/LazyVim/LazyVim/commit/7a1414b5e5950bde34b9bdc0dbc3dd20109f2ac9) with this change. It seems copilot-chat deprecated these options and that's why they were changed in Lazyvim as well. For more info about the deprecated changes you should inquire at copilot-chat's repo.
